wagtail/tests/search/migrations/0001_initial.py

```diff
@@ -54,7 +54,7 @@ class Migration(migrations.Migration):
             name='ProgrammingGuide',
             fields=[
                 ('book_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='searchtests.Book')),
-                ('programming_language', models.CharField(choices=[('py', 'Python'), ('js', 'Javascript'), ('rs', 'Rust')], max_length=255)),
+                ('programming_language', models.CharField(choices=[('py', 'Python'), ('js', 'JavaScript'), ('rs', 'Rust')], max_length=255)),
```

Comment:
Do we want to change migrations? I'm fine as choices don't trigger db changes. Just wanted to mention it.

Reply:
These are just test migrations and this change doesn't actually alter schema anyway